public/404.html

```diff
@@ -1,67 +1,83 @@
-  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <head>
+  <meta charset=utf-8>
+  <meta content="width=device-width,initial-scale=1.0,minimum-scale=1.0,maximum-scale=1.0,user-scalable=no" name=viewport>
+  <title>Application error | DAHLIA San Francisco Housing Portal </title>
+  <link rel="icon" type="image/png" href="/images/favicon/favicon-32x32.png" sizes="32x32">
   <style>
-  body {
-    background-color: #EFEFEF;
-    color: #2E2F30;
-    text-align: center;
-    font-family: arial, sans-serif;
-    margin: 0;
-  }
+    @import url('https://fonts.googleapis.com/css?family=Droid+Serif');
+
+    html, body {
+      font-family: 'Droid Serif', sans-serif;
+      -ms-text-size-adjust: 100%;
+      -webkit-text-size-adjust: 100%;
+      background-color: #FFF;
+      height: 100%;
+      -webkit-font-smoothing: antialiased;
+    }
 
-  div.dialog {
-    width: 95%;
-    max-width: 33em;
-    margin: 4em auto 0;
-  }
+    body {
+      margin: 0;
+      padding: 0;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+      border-top: 5px solid #0077da;
+    }
 
-  div.dialog > div {
-    border: 1px solid #CCC;
-    border-right-color: #999;
-    border-left-color: #999;
-    border-bottom-color: #BBB;
-    border-top: #B00100 solid 4px;
-    border-top-left-radius: 9px;
-    border-top-right-radius: 9px;
-    background-color: white;
-    padding: 7px 12% 0;
-    box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
-  }
+    .logo {
+      position: absolute;
+      top: 0;
+      box-shadow: 0px 1px 2px 0px rgba(0,0,0,0.25);
+    }
 
-  h1 {
-    font-size: 100%;
-    color: #730E15;
-    line-height: 1.5em;
-  }
+    .message {
+      text-align: center;
+      align-self: center;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      padding: 0px 20px;
+      max-width: 450px; }
 
-  div.dialog > p {
-    margin: 0 0 1em;
-    padding: 1em;
-    background-color: #F7F7F7;
-    border: 1px solid #CCC;
-    border-right-color: #999;
-    border-left-color: #999;
-    border-bottom-color: #999;
-    border-bottom-left-radius: 4px;
-    border-bottom-right-radius: 4px;
-    border-top-color: #DADADA;
-    color: #666;
-    box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
-  }
-  </style>
-</head>
+    .message__title {
+      font-size: 24px;
+      font-weight: 100;
+      margin-top: 15px;
+      color: #47494E;
+      margin-bottom: 8px; }
 
-<body>
-  <!-- This file lives in public/404.html -->
-  <div class="dialog">
-    <div>
-      <h1>The page you were looking for doesn't exist.</h1>
-      <p>You may have mistyped the address or the page may have moved.</p>
+    p {
+      -webkit-margin-after: 0px;
+      -webkit-margin-before: 0px;
+      font-size: 15px;
+      color: #7F828B;
+      line-height: 21px;
+      margin-bottom: 4px; }
+    }
+    </style>
+  </head>
+  <body><script id="__bs_script__">//<![CDATA[
```

Comment:
@kaptron @mluedke2 This 404 page looks nicer! I don't think we need this browser sync markup, though.